easybuild/easyconfigs/h/h5py/h5py-2.4.0-intel-2015a-Python-2.7.9-serial.eb

```diff
@@ -29,7 +29,7 @@ dependencies = [
 builddependencies = [('pkgconfig', '1.1.0', '-Python-%(pyver)s')]
 
 sanity_check_paths = {
-    'files': ['lib/python%s/site-packages/%%(name)s-%%(version)s-py%s-linux-x86_64.egg' % (pythonshortver, pythonshortver)],
+    'files': ['lib/python%s/site-packages/%%(name)s-%%(version)s-py%s-linux-x86_64.egg' % (2 * (pythonshortver, ))],
```

Comment:
use `%(pyshortver)s` instead? and don't explicitely check for the `.egg`, move it to `dirs`

also drop the `pyshortver = ` line